src/mysql.jl

```diff
@@ -1,5 +1,8 @@
-function _dbconnect(conn_obj::Type{MySQL.Connection}, kwargs)
+function _dbconnect(conn_obj::Type{MySQL.Connection}, host::String, user::String, password::String; db::String="", port::Integer=3306, unix_socket::Union{Nothing,String}=nothing, client_flag=API.CLIENT_MULTI_STATEMENTS, opts = Dict())
```

Comment:
Hm, I am not too sure on this layout here; I think it would be better to have the second argument actually only be a string that is passed to the connection. Let's discuss during our check-in.